README.md

```diff
@@ -2,6 +2,12 @@
 
 A simple React application for managing music tracks, built with Vite and Material-UI.
 
+## Architecture Decision Records (ADRs)
+
+This project uses Architecture Decision Records (ADRs) to document important architectural decisions made along with their context and consequences.
+
+- [ADR-0001: TypeScript Adoption](docs/adr/0001-typescript-adoption.md)
```

Comment:
I suggest to categorize ADRs by sections/features/domains to prevent possible mess in future when new documentations will be added (which may relate to different features).
If all ADRs will be stored as a plain list it is good for historical tracking of all progress, but may be hard for developers to find specific docs which they may need.
No strong opinion though